v1/BetterRTX_Installer.ps1

```diff
@@ -294,10 +296,37 @@ Switch ($selection) {
         $version = $releases[$SelectVersion - 1]
         $newStubUrl = $version.stub
         $newToneMappingUrl = $version.tonemapping
+
+        #   TODO: Host a separate CDN for older tonemappings.
+        #         Currently, the default tonemapper is used for all variations of BetterRTX
+
+        $oldToneMappingUrl = "https://cdn.discordapp.com/attachments/1111388664468086836/1148375947129663528/RTXPostFX.Tonemapping.material.bin?ex=663c9db1&is=663b4c31&hm=afb0a9f03211f6ce9ed004207410a2543b32437be06d8c9dbc1b12b95f49ebe1&"
+
         Write-Host ""
         Write-Host $lang.downloadingBins
         Invoke-WebRequest -URI $newStubUrl -OutFile $newStub -UseBasicParsing;
-        Invoke-WebRequest -URI $newToneMappingUrl -OutFile $newTonemapping -UseBasicParsing;
+
+        # Install an older tonemapping for older Minecraft Versions
+        [xml]$minecraftManifest = Get-Content -Path $manifestLocation
+        $installationVersion = $minecraftManifest.Package.Identity.Version.Substring(0, 7).Split(".")
+        $releaseVer = [int]$installationVersion[0]
+        $majorVer = [int]$installationVersion[1]
+        $minorVer = [int]$installationVersion[2]
```

Comment:
It'd be better to either split this into a separate function for clarity or label what each part is doing.